### PR TITLE
SRM Glaive-Class Hunter Ship

### PR DESCRIPTION
**Pull request by Latentish**
## About The Pull Request
The SRM has no unique ship at the moment. This adds a ship to this unrepresented PMC faction that is extremely lore accurate.

glaive

This ship is a love letter to the Saint-Roumain Militia. This ship features: heavy influence to invest in tribal crafting, late game gun manufacturing research nodes, a glade of dirt from the homeworld of (insert name of the hunter who bought the ship here), animals and a smidge of raw meat to encourage their usual (religiously inspired) meal habits of raw or charred meats, very basic farming to reinforce tribal crafting and tribal medicine as well as feeding those who cannot eat meat, semi-decently equipped mining, and a religious hierarchy. I think this ship will have a similar effect on players to the Lamia, in that it could encourage people to actually roleplay the culture through heavy theming.
image
image
image

## Why It's Good For The Game
The SRM are very lore rich, are interesting, and have a culture that would be fun to emulate. This ship is a huge cultural piece that reads easily to anyone who joins it. It's very easy to see what the SRM is all about just by looking around. This is the only ship for the SRM that is not a reskin of another ship.

## Changelog
:cl:phoaly
add: Glaive-class SRM ship added
/:cl:

### CODE REVIEW

**Comment by Latentish** [comment]
image
bloodborne references

**Comment by RuneQuester** [comment]
> image bloodborne references

Oh dear, what was it? The lag, the griefers, or the horrible maps?
Oh, it doesn't matter. It always comes down to the admins to clean up after these sorts of messes

**Comment by Latentish** [comment]
This should be test merged when the event to introduce the SRM is ready. Once that happens, I'll make the ship purchaseable, and we can get it actually tested enough to merge. So DNM for now.

**Comment by Latentish** [comment]
> This PR has been inactive for long enough to be automatically marked as stale. This means it is at risk of being auto closed in ~ 14 days, please address any outstanding review items and ensure your PR is finished, if these are all true and you are auto-staled anyway, you need to actively ask maintainers if your PR will be merged. Once you have done any of the previous actions then you should request a maintainer remove the stale label on your PR, to reset the stale timer. If you feel no maintainer will respond in that time, you may wish to close this PR youself, while you seek maintainer comment, as you will then be able to reopen the PR yourself

IT IS LITERALLY TM'D 
Die

**Comment by Latentish** [comment]
I'm FUCKING PISSED.

**Comment by ArdentSpark** [comment]
Haha oh my god that reason for the CI fail is amusing. I had a suggestion, may I propose that these airalarms and APC's bordering the 'secret rooms' are moved? That way the option of opening up the walls and expanded into from the locker room and medical room is there, without having to move machinery

**Comment by ArdentSpark** [comment]
The Winchester Mk2 Rifle added by this PR to the techweb is unprintable, due to requiring Wood, which is not able to be added as a material to Circuit printers, Protolathes, Autolathes, ORMs or Silos due to not containing sufficient materials

**Comment by Latentish** [comment]
I'll finish this and mark it ready for review later

**Review by PrefabQuasar — CHANGES_REQUESTED** [review]
this map doesnt seem to have any major issues, nor have any shown up when it was TM'd, however, i'd reccomend either changing the wardrobe lockers to another sprite or making them not access locked, as the sprite currently used isnt designed to have an access lock, so it looks odd on them, i'm also unsure on the inclusion of sentience potions in here, i feel like too often those can just lead to "animal that acts like a human in every respect", which doesnt feel great for RP

**Comment by PrefabQuasar** [comment]
> The Winchester Mk2 Rifle added by this PR to the techweb is unprintable, due to requiring Wood, which is not able to be added as a material to Circuit printers, Protolathes, Autolathes, ORMs or Silos due to not containing sufficient materials  

assuming that this has been addressed

**Comment by Latentish** [comment]
> assuming that this has been addressed

Yes this has been addressed and the wood has been removed from the requirement.

**Comment by Latentish** [comment]
> this map doesnt seem to have any major issues, nor have any shown up when it was TM'd, however, i'd reccomend either changing the wardrobe lockers to another sprite or making them not access locked, as the sprite currently used isnt designed to have an access lock, so it looks odd on them, i'm also unsure on the inclusion of sentience potions in here, i feel like too often those can just lead to "animal that acts like a human in every respect", which doesnt feel great for RP

I can swap the locker sprite.

The sentience potion here is for part of the ship's niche of beast taming. I get that players might act like normal people as sentient animals, but I feel that's a reflection on sentience potions as a whole. I am unsure if there's another potion to let players take over simples.

**Comment by PrefabQuasar** [comment]
> > this map doesnt seem to have any major issues, nor have any shown up when it was TM'd, however, i'd reccomend either changing the wardrobe lockers to another sprite or making them not access locked, as the sprite currently used isnt designed to have an access lock, so it looks odd on them, i'm also unsure on the inclusion of sentience potions in here, i feel like too often those can just lead to "animal that acts like a human in every respect", which doesnt feel great for RP
> 
> I can swap the locker sprite.
> 
> The sentience potion here is for part of the ship's niche of beast taming. I get that players might act like normal people as sentient animals, but I feel that's a reflection on sentience potions as a whole. I am unsure if there's another potion to let players take over simples.

i'm still iffy on them, i'd want a third opinion really

**Comment by Apogee-dev** [comment]
> i'm still iffy on them, i'd want a third opinion really

Sentience potions are fine imo due to mechanical need. The beast tamer gimmick just plain doesn't work without them, and it gives ghosts more stuff to do.

**Comment by PrefabQuasar** [comment]
good enough for me, although one day i'd hope to see a potion that gives sentience but not the common language. actually that might not be too hard to add

**Review by PrefabQuasar — APPROVED** [review]
regardless, thats everything i had adressed, and nothing's come up during testing

File: code/game/MapData/shuttles/srm_glaive.dm [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+	var/static/items_inside = list(
+		/obj/item/healthanalyzer = 1,
+		/obj/item/reagent_containers/food/snacks/grown/ash_flora/puce = 1,
+		/obj/item/reagent_containers/glass/mortar = 1,
+		/obj/item/reagent_containers/glass/bowl/mushroom_bowl = 1,
+		/obj/item/pestle = 1,
+		/obj/item/reagent_containers/food/snacks/grown/ash_flora/cactus_fruit = 3,
+		/obj/item/reagent_containers/food/snacks/meat/slab/bear = 3,
+		/obj/item/reagent_containers/food/snacks/grown/ash_flora/mushroom_leaf = 3)
+	generate_items_inside(items_inside,src)
```
Comment (Fikou) [inline code/game/MapData/shuttles/srm_glaive.dm]:
```suggestion
	var/static/list/items_inside = list(
		/obj/item/healthanalyzer = 1,
		/obj/item/reagent_containers/food/snacks/grown/ash_flora/puce = 1,
		/obj/item/reagent_containers/glass/mortar = 1,
		/obj/item/reagent_containers/glass/bowl/mushroom_bowl = 1,
		/obj/item/pestle = 1,
		/obj/item/reagent_containers/food/snacks/grown/ash_flora/cactus_fruit = 3,
		/obj/item/reagent_containers/food/snacks/meat/slab/bear = 3,
		/obj/item/reagent_containers/food/snacks/grown/ash_flora/mushroom_leaf = 3,
	)
	generate_items_inside(items_inside, src)
```

**Pull request merged**